src/main/java/org/distributed/systems/chord/actors/MemcachedActor.java

```diff
@@ -0,0 +1,124 @@
+package org.distributed.systems.chord.actors;
+
+import akka.actor.AbstractActor;
+import akka.actor.ActorRef;
+import akka.event.Logging;
+import akka.event.LoggingAdapter;
+import akka.io.Tcp;
+import akka.io.TcpMessage;
+import akka.pattern.Patterns;
+import akka.util.ByteString;
+import org.distributed.systems.chord.messaging.KeyValue;
+import scala.concurrent.Await;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.Arrays;
+
+import scala.concurrent.Future;
+import akka.util.Timeout;
+
+class MemcachedActor extends AbstractActor {
+
+    private final LoggingAdapter log = Logging.getLogger(getContext().getSystem(), this);
+    private static final int STANDARD_TIME_OUT = 1000;
```

Comment:
Maybe define this for the entire application. (I thought this was already in ChordStart)